packages/api/src/controllers/asset.ts

```diff
@@ -328,7 +339,11 @@ app.get("/", authorizer({}), async (req, res) => {
     res.links({ next: makeNextHREF(req, newCursor) });
   }
 
-  return res.json(output);
+  if (details) {
+    return res.json(output);
+  } else {
+    return res.json(cleanAssetResponse(output));
+  }
```

Comment:
I think this should be done on that same [`mung` middleware](https://github.com/livepeer/livepeer-com/blob/98652ad082a738f3697e17862192994946786184/packages/api/src/controllers/asset.ts#L212). This will allow it to:
 - Clean the asset objects in every API, not only in the list here (we want that, right?)
 - Keep any response-cleaning logic centralized in that function, so there's only one place to look (even if it has some internal helper, there's still only one entry-point for that)

WDYT?

Reply:
Bump! Do you think this would take too much time for you? If so, I think it's ok like this and I can move it to `mung` on https://github.com/livepeer/studio/pull/1178 where I'm already doing some changes here (then you can review the PR 👼)

Reply:
Moved it into the `mung` middleware! It works, not sure it's the best / right way to do that